source/zh-tw/docs/helpers.md

```diff
@@ -316,6 +316,15 @@ title: 輔助函數（Helpers）
 <%- render(str, engine, [options]) %>
 ```
 
+**範例：**
+
+``` js
+<%- render('p(class="example") Test', 'pug'); %>
+// <p class="example">Test</p>
+```
+
+有關更多詳細信息，請參見[渲染](https://hexo.io/zh-tw/api/rendering.html)。
```

Comment:
As far as I know, taiwanese uses `資訊` instead of `信息`.

IMHO, you could just copy the original English sentences to other languages and it will be fine. Translation should be done by volunteer, not us.

Reply:
Sure, next time I'll submit the english content in all languages sections then contributors may be able to translate it if they want to rather than me using google tranlaste and ending with bad quality sentences.